### PR TITLE
Register Subdomain: aqua

### PR DESCRIPTION
**Pull request by AquaQuokka**
Added the domain: aqua

### CODE REVIEW

**Comment by WilliamDavidHarrison** [comment]
You only need aqua.json

**Comment by AquaQuokka** [comment]
Wdym?

**Comment by WilliamDavidHarrison** [comment]
> Wdym?

In this PR you have made 2 files. `aqua.json` and just `aqua`. please delete the file that doesn't end in `.json`

**Comment by AquaQuokka** [comment]
Oh ok

**Comment by AquaQuokka** [comment]
Where is it?

**Comment by AquaQuokka** [comment]
Done! @WilliamDavidHarrison

**Comment by AquaQuokka** [comment]
HOld on can you help me delete it? It won't let me?

**Comment by AquaQuokka** [comment]
oh nvm it's gone

**Review by wdhdev — DISMISSED** [review]

**Comment by AquaQuokka** [comment]
Now what do I do

**Review by MaskDuck — DISMISSED** [review]

**Review by AquaQuokka — COMMENTED** [review]
a

File: domains/aqua.json
Comment (AquaQuokka) [reply domains/aqua.json:9]: Can you rephrase @MaskDuck ?

**Comment by mtgsquad** [comment]
Dude, URL means it's going to redirect to that URL, it doesn't register aqua.is-a.dev, it will in fact redirect to it, redirecting to itself is an invalid record. please switch to a CNAME or A record to point towards a web app/website/server.

File: domains/aqua.json
Comment (AquaQuokka) [reply domains/aqua.json]: @mtgsquad Done

File: domains/aqua.json
Comment (WilliamDavidHarrison) [inline domains/aqua.json:9]: Please add `https://` in front of the URL.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/aqua.json
Comment (AquaQuokka) [reply domains/aqua.json:9]: Ok

Comment (AquaQuokka) [reply domains/aqua.json]: @WilliamDavidHarrison Done.

**Review by wdhdev — APPROVED** [review]

**Comment by AquaQuokka** [comment]
Now what? @WilliamDavidHarrison ?

**Comment by WilliamDavidHarrison** [comment]
> Now what? @WilliamDavidHarrison ?

Wait for maintainer to merge it.

**Comment by AquaQuokka** [comment]
How long will that take?

**Comment by WilliamDavidHarrison** [comment]
> How long will that take?

Maybe a few days.

**Comment by AquaQuokka** [comment]
@WilliamDavidHarrison Problem is, this request is so old that no one will find it. 😆

**Comment by WilliamDavidHarrison** [comment]
> @WilliamDavidHarrison Problem is, this request is so old that no one will find it. 😆 

Maintainers normally merge the oldest PRs first.

**Comment by AquaQuokka** [comment]
*awkward silence*

**Review by TheBotlyNoob — APPROVED** [review]

**Pull request merged**
